salt/modules/cmdmod.py

```diff
@@ -589,11 +589,17 @@ def _get_stripped(cmd):
             out = proc.stdout.decode(__salt_system_encoding__)
         except AttributeError:
             out = u''
+        except UnicodeDecodeError:
+            log.error("UnicodeDecodeError while decoding output of cmd {0}".format(cmd))
+            out = proc.stdout.decode(__salt_system_encoding__, 'ignore')
```

Comment:
replace might be better than ignore. At least the user will be more aware of it. Might be worth adding the ability to override `__salt_system_encoding__`

Reply:
I like replace. Changed and rebased